### DISCUSSION TITLE
Best place to file npmjs.com issues

### DISCUSSION THREAD

**Discussion by sguillia** [npm/feedback#505] [Registry] [answered]
Hi

There is an issue on the npmjs.com website that prevents granting write access to a team.
Because of this issue, my company has lost write access to a package.
I have contacted the npm support on November 15th, 2020 but unfortunately they never replied.

I apologize as these discussions are not the best place to talk about this, but I found no better option on https://npm.community/ because this is not a bug with npm cli, and npm support doesn't answer.

A screenshot of the issue after clicking Read/Write can be found [here](https://user-images.githubusercontent.com/15984272/127132494-25c5b867-3498-4e38-8bac-cb55a2f3dbbc.png).

Do you have any idea on how I should go about this?
Thank you

**Comment by MylesBorins** [comment, marked as answer]
@sguillia thanks for opening this discussions. Apologies that you were not able to get support for the ticket you opened in November. We've made improvements to how we handle support since last year, would you be able to open a new ticket via https://npmjs.com/support and let me know. I'll escalate it internally

**Reply by sguillia** [reply]
@MylesBorins The issue seems solved after a new exchange with the npm support, thanks so much for your quick reply and for your help
